lib/requester/request-wrapper.js

```diff
@@ -86,13 +90,50 @@ var _ = require('lodash'),
               options.proxy = proxyConfig.server.toString(),
               options.tunnel = proxyConfig.tunnel;
               _.set(options, 'request._._postman_proxy', proxyConfig);
+              _.set(options, 'request.proxy', proxyConfig);
             }
             return cb(null, options);
         }
         cb(null, options);
     },
 
     /**
+     * Resolves the content of a certificate from the file system,
+     * using the current file resolver.
+     *
+     * @param certificate The certificate to resolve
+     * @param resolver The file resolver to use
+     * @param cb Callback once resolve is done
+     */
+    resolveCertificateContents = function(certificate, resolver, cb) {
+        var keyPath = _.get(certificate, 'key.src'),
+            certPath = _.get(certificate, 'cert.src'),
+            keyData,
+            certData;
+
+        if (!_.isString(keyPath) || !_.isString(certPath)) {
+            cb(ERROR_INVALID_FILE_PATH);
+        }
+
+        async.waterfall([
```

Comment:
Why use waterfall for this?

You can do something like async.mapParallel with [keyPath, certPath], resolver.readFile, function () …